setup.py

```diff
@@ -1,15 +1,29 @@
 from setuptools import setup
 
-setup(name='cromwell-tools',
-      version='1.0.0.dev1',
-      description='Utilities for interacting with the Cromwell workflow engine',
-      url='http://github.com/broadinstitute/cromwell-tools',
-      author='Mint Team',
-      author_email='mintteam@broadinstitute.org',
-      license='BSD 3-clause "New" or "Revised" License',
-      packages=['cromwell_tools'],
-      install_requires=[
-          'requests'
-      ],
-      include_package_data=True
-      )
+CLASSIFIERS = [
+    "Development Status :: 4 - Beta",
+    "Natural Language :: English",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 2.7",
```

Comment:
should add this into README if crowmwell-tool only work with python2.7

Reply:
Great idea; I think it may not be necessary to support python2.7 only; I'll take a quick look, and will document it in the README if it's the case.

Reply:
Added python3 support.